tests/CodeGen-MkBundle/Xamarin.Android.MakeBundle-UnitTests/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
```

Comment:
What is this file used for?  I don't see any other mentions of `run.sh`.

Reply:
it's just a way to run this without having to launch the whole msbuild machinery - faster turnover of changes